src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import ball.characteristics.Ball;
+import ball.characteristics.Color;
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        Ball ball = new Ball();
+        ball.setColor(Color.values()[index].toString());
+        return ball.getColor();
```

Comment:
Instead of using `toString()`, use `name()` for getting the String representation of enum constants to avoid issues if `toString()` is overridden in the enum.

Reply:
not fixed